### PR TITLE
Feature/create board by workspace

### PR DESCRIPTION
**Pull request by albcl**
### What changed:
Added `create_boards()` function and a test for it.
Also, `Readme` has been updated.

### What resolves:
Issue #59: Will allow users to easily create boards optionally specifying a workspace for it.

### CODE REVIEW

File: monday/query_joins.py [hunk 1]
```diff
@@ -356,6 +356,18 @@ def get_columns_by_board_query(board_ids):
         }''' % board_ids
 
 
+def create_board_by_workspace_query(board_name, board_kind, workspace_id = None):
```
[the remark this answers is not in the document]
Comment (chdastolfo) [reply monday/query_joins.py:359]: Agreed.

**Review by chdastolfo — DISMISSED** [review]

**Comment by chdastolfo** [comment]
Adding type hints as @rhymiz requested is non-blocking, but would prefer them to be added before merge @albcl. Would prefer to merge both of your PRs at the same time, if possible.

File: monday/query_joins.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rhymiz) [reply monday/query_joins.py:359]: pinging @albcl 😄

**Comment by albcl** [comment]
Updated with type hints and some custom ones

**Comment by albcl** [comment]
I think this should be ready to review now 😄

**Comment by rhymiz** [comment]
Thanks again @albcl, will take a look!

**Review by rhymiz — APPROVED** [review]

**Comment by rhymiz** [comment]
@all-contributors please add @albcl for code

**Comment by rhymiz** [comment]
@all-contributors please add @albcl to code

**Comment by rhymiz** [comment]
@all-contributors please remove @albcl from code

**Pull request merged**